corehq/ex-submodules/soil/util.py

```diff
@@ -94,16 +95,18 @@ def get_download_context(download_id, message=None, require_result=False):
 
 def process_email_request(download_id, email_address):
     dropbox_url = absolute_reverse('dropbox_upload', args=(download_id,))
+    download_url = "{}?get_file".format(absolute_reverse('retrieve_download', args=(download_id,)))
     try:
         allow_dropbox_sync = get_download_context(download_id).get('allow_dropbox_sync', False)
     except TaskFailedError:
         allow_dropbox_sync = False
-    download_url = "{}?get_file".format(absolute_reverse('retrieve_download', args=(download_id,)))
-    email_body = 'Your CommCare export is ready! Click on the link below to download your ' + \
-                 'requested data:\n' + download_url
-    dropbox_message = '\n\nYou can also upload your data to Dropbox with the link below:\n' + \
-                      dropbox_url if allow_dropbox_sync else ''
-    send_HTML_email('CommCare Export Complete', email_address, email_body + dropbox_message)
+    dropbox_message = ''
+    if allow_dropbox_sync:
+        dropbox_message = ('\n\nYou can also upload your data to Dropbox with the link below:\n'
+                           '{}').format(dropbox_url)
+    email_body = ('Your CommCare export is ready! Click on the link below to download your requested data:\n'
```

Comment:
I think you didn't actually translate these? You need to use `_` the way you did below.